cfgov/jinja2/v1/_includes/organisms/mega-menu-var-2.html

```diff
@@ -193,6 +223,7 @@
                 {% endif %}
             </div>
             {% endif %}
+            {% endif %}
```

Comment:
Is this double `endif` correct? If so, should the inside block be indented one level?

Reply:
It is. The first `endif` is from [this block](https://github.com/cfpb/cfgov-refresh/blob/3f564b25bd5228122304ee931c70fb84ecfada23/cfgov/jinja2/v1/_includes/organisms/mega-menu-var-2.html#L211-L225), one of the hardcoded ones you added. I could and maybe should have indented but didn't want to add so many more diffs to these files, especially since they're still just proposed variations.